.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
     -   id: pyupgrade
         args: [--py3-plus]
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.740
+    rev: v0.740  # NOTE: keep this in sync with setup.py.
```

Comment:
Just for anyone doing the merge from features, note that features is already at `v0.761`.